src/lib.rs

```diff
@@ -1384,11 +1384,64 @@ pub fn logger() -> &'static Log {
 
 // WARNING: this is not part of the crate's public API and is subject to change at any time
 #[doc(hidden)]
+#[cfg(not(feature = "kv_unstable"))]
 pub fn __private_api_log(
     args: fmt::Arguments,
     level: Level,
     &(target, module_path, file, line): &(&str, &'static str, &'static str, u32),
+    kvs: Option<&[(&str, &str)]>,
```

Comment:
We should use `&ToValue` here so that the structure of captured values can be retained. I sketched out the change in https://github.com/yoshuawuyts/kv-log-macro/pull/1

Reply:
Ohhhh, yesss! -- I wish I'd known about this sooner, haha.